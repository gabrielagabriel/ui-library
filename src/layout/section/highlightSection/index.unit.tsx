import React from 'react'
import renderer from 'react-test-renderer'

import HighlightSection from './index'

describe('HighlightSection', () => {
  it('should render default highlight section', () => {
    const section = <HighlightSection title="the title">default highlight section</HighlightSection>
    const renderedSection = renderer.create(section).toJSON()
    expect(renderedSection).toMatchSnapshot()
  })

  it('should add a classname to the content', () => {
    const section = (
      <HighlightSection title="the title" contentClassName="some-content-class-name">
        default highlight section
      </HighlightSection>
    )
    const renderedSection = renderer.create(section).toJSON()
    expect(renderedSection).toMatchSnapshot()
  })
})
