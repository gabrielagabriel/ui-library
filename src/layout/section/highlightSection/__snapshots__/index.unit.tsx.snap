// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighlightSection should add a classname to the content 1`] = `
.c1 .section-content {
  padding-left: 24px;
  padding-right: 24px;
}

.c3 {
  color: #054752;
  font-size: 30px;
  font-weight: 500;
  line-height: 1.06;
}

.c2 {
  font-size: 22px;
  line-height: 24px;
  padding-top: 24px;
  padding-bottom: 8px;
  margin: 0;
}

.c0 {
  background-color: #5DD167;
  padding-bottom: 24px;
}

.c0 .section-content {
  position: relative;
}

.c0 .kirk-subheader {
  color: #FFF;
  font-size: 30px;
  padding-top: 24px;
  padding-bottom: 24px;
}

@media (min-width:800px) {
  .c1 .section-content {
    margin-left: auto;
    margin-right: auto;
    max-width: 662px;
  }

  .c1 .section-content.section-content--large {
    max-width: 1016px;
  }
}

@media (max-width:800px) {
  .c0 .kirk-subheader {
    text-align: center;
  }
}

<div
  className="c0 c1"
  role="presentation"
>
  <article
    className="section-content some-content-class-name section-content--large"
  >
    <h2
      className="kirk-title kirk-subheader c2 c3"
    >
      the title
    </h2>
    default highlight section
  </article>
</div>
`;

exports[`HighlightSection should render default highlight section 1`] = `
.c1 .section-content {
  padding-left: 24px;
  padding-right: 24px;
}

.c3 {
  color: #054752;
  font-size: 30px;
  font-weight: 500;
  line-height: 1.06;
}

.c2 {
  font-size: 22px;
  line-height: 24px;
  padding-top: 24px;
  padding-bottom: 8px;
  margin: 0;
}

.c0 {
  background-color: #5DD167;
  padding-bottom: 24px;
}

.c0 .section-content {
  position: relative;
}

.c0 .kirk-subheader {
  color: #FFF;
  font-size: 30px;
  padding-top: 24px;
  padding-bottom: 24px;
}

@media (min-width:800px) {
  .c1 .section-content {
    margin-left: auto;
    margin-right: auto;
    max-width: 662px;
  }

  .c1 .section-content.section-content--large {
    max-width: 1016px;
  }
}

@media (max-width:800px) {
  .c0 .kirk-subheader {
    text-align: center;
  }
}

<div
  className="c0 c1"
  role="presentation"
>
  <article
    className="section-content section-content--large"
  >
    <h2
      className="kirk-title kirk-subheader c2 c3"
    >
      the title
    </h2>
    default highlight section
  </article>
</div>
`;
